README.md

```diff
@@ -1,105 +1,88 @@
+If you _are_ done, you can follow [this procedure](./gh-pages-procedure.md) to get your project live on GitHub Pages.
 
----
+Visit the site at [GitHub Pages](https://pages.github.com/)
 
-## Test Files Organization
+Otherwise, in order to deploy this site locally, a user will have to run a local server.
 
-Similar to feature code, your test code needs to be put in a specific place for it to run successfully.
+  1. Clone (or fork and then clone) this repo with  `git clone git@github.com:JeffKersting/refactor-tractor-whats-cookin.git [what you want to name the repo]`
+  2. Change into the directory and run `npm install` to install project dependencies.
+  3. Run `npm start` to start a local server 
+  4. To access the site in your browser, navigate to the localhost address that your terminal gives you. 
+  5. For any of the class tests, run ```npm test [name of the test you want to run]``` 
+  6. Enter `control + c` in your terminal to stop the server at any time.
 
-**Put all of your test files in the `test` directory.** As a convention, all test filenames should end with `-test.js`. For instance: `box-test.js`.
+## Contributors
 
-## Running Your Tests
+[This project](https://frontend.turing.io/projects/module-2/refactor-tractor-wc.html) was used exisiting code from previous students and staff at [Turing School Staff](https://turing.io/)
 
-Run your test suite using the command:
+This refactor was coded by [Jeff Kersting](https://github.com/JeffKersting), [Allison Dietz](https://github.com/dietza), [Thao Ma](https://github.com/thaomonster) and [Alice Ruppert](https://github.com/srslie).
 
-```bash
-npm test
-```
+Special thanks to our mentors [Scott Schipke](https://github.com/sschipke) 
 
-The test results will output to the terminal.
+//ADD MENTORS?
 
----
+## Technologies
 
-## Linting Your Code
+During this project we gained experience with the following technologies:
+- ES6 JavaScript
+- Testing with Mocha and Chai
+- Linting with ESLint
+- Semantic HTML
+- Responsive SCSS with Media Queries
+- Wave, ColorBlind, and ARIA Accessibility
+- Using network requests with API endpoints and async JavaScript
+- Using Webpack to compile mulitple class and script files
+- Deploying a site on GitHub Pages
+- Git workflow using GitHub Issues and Project Kanban Board
 
-Run the command in your terminal `npm run lint` to run the linter on your JavaScript code. There will be errors and warnings right from the start in this starter kit - the linter is still running successfully.
+We also gained invaluable experience working with a larger group (4 people) and collaborating on exisiting data.
+## Future Iterations
 
-Your linter will look at the JavaScript files you have within the `src` directory and the `test` directory. 
+This project helped us practice familiarizing ourselves with exisiting code, working asynchronously with three others a project in a relatively short time-frame, and honed our planning, communication and project managament.
```

Comment:
it feels like this could fit into the section above - more like the 'learning goals'?

Reply:
Moved!